### ISSUE TITLE
Users in private servers using the beestation/beestation docker image cannot use vending machines

### ISSUE THREAD

**Issue by cw0** [BeeStation/BeeStation-Hornet#2585]
## Reproduction:
Create a server from a docker image and connect as a Host user. Even after deadmining you cannot interact with vending machines. Also affects non privileged users. I have tested from 2 separate computers both in windows and a wine wrapper.

**Comment by Crossedfall** [comment]
Problem seems to impact a couple of TGUI interfaces.

**Comment by cw0** [comment]
I have tried to increment the beestation/byond image versions in the beestation/beestation docker image as well but there is no difference. Running the codebase locally using the Dream Daemon works fine but I'm not sure what the differences between the two run methods are. I also noticed trying to write on paper in game also does not work correctly. My key bindings are also unset when attempting to join.

For replication sake simply running:
`docker run -p 1337:1337 beestation/beestation`
is enough to replicate the issue. Adding volumes for config just enables admin powers, still has the same issues.
I am using byond version 513.1532
